.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can you please add the line `.vscode` to `.gitignore` in the root directory of smag-mvp?